language/move-vm/runtime/src/special_names.rs

```diff
@@ -11,3 +11,5 @@ pub(crate) static EMIT_EVENT_NAME: Lazy<Identifier> =
 
 pub(crate) static SAVE_ACCOUNT_NAME: Lazy<Identifier> =
     Lazy::new(|| Identifier::new("save_account").unwrap());
+
+pub(crate) static PRINT_NAME: Lazy<Identifier> = Lazy::new(|| Identifier::new("print").unwrap());
```

Comment:
is this needed at all? I ask because I don't see any usage and I would expect no usage to be there

Reply:
No it's not. Forgot to delete it.